assets/component-menu-drawer.css

```diff
@@ -224,11 +224,20 @@ details[open].menu-opening > .menu-drawer__submenu {
   align-items: center;
   text-decoration: none;
   padding: 1.2rem;
-  margin-left: -1.2rem;
   font-size: 1.4rem;
   color: rgb(var(--color-foreground));
 }
 
+@media screen and (min-width: 750px) {
+  .menu-drawer__account {
+    display: none;
+  }
+
+  .menu-drawer__utility-links:has(ul:empty) {
+    display: none;
+  }
+}
+
```

Comment:
I think we could remove the margin about the social media list if it's an `:only-child` in the drawer: 
<details>
<summary>Screenshot</summary>

![](https://screenshot.click/17-11-7l8om-b97zk.png)
</details>

Reply:
ah yeah I realize now 👍

Reply:
The only workaround I could find is to use `:has`, move margin from `.list-social` to .menu-drawer__account and change it to margin-bottom.

I saw you feedback about `:has` but I think it's okay and at least we could improve it for those cases when `:has` is supported.